R/FilteredDataset.R

```diff
@@ -407,6 +423,15 @@ FilteredDataset <- R6::R6Class( # nolint
           logger::log_trace("FilteredDataset$server initializing, dataname: { deparse1(dataname) }")
           checkmate::assert_string(dataname)
           shiny::setBookmarkExclude("remove_filters")
+
+          output$filter_count <- renderText(
+            sprintf(
+              "%d filter%s applied",
+              length(self$get_filter_state()),
```

Comment:
so in MAE case this isn't quite right:

![image](https://user-images.githubusercontent.com/15201933/210582314-adeb1414-d8d3-4772-8c98-ef60e0a95da6.png)

![image](https://user-images.githubusercontent.com/15201933/210582423-4827f409-aff3-4c3a-9131-ea0ae954a819.png)

Reply:
Maybe just leave a "filter applied" darkred info? It might be also misleading in when the filter was added, but not actual filtering was done (nothing selected).

Reply:
I could change "applied" to "selected" ?

> can we have this info also after Active Filter Variables collapse?

I'm going to defer that to #6